apps/omg_child_chain_rpc/priv/swagger/operator_api_specs.yaml

```diff
@@ -90,16 +83,31 @@ paths:
                       data:
                         blknum: 123000
                         txindex: 111
-                        txhash: >-
-                          0xbdf562c24ace032176e27621073df58ce1c6f65de3b5932343b70ba03c72132d
-        '500': &ref_1
+                        txhash: '0xbdf562c24ace032176e27621073df58ce1c6f65de3b5932343b70ba03c72132d'
+        '500':
           description: Returns an internal server error
           content:
             application/json:
               schema:
                 description: The response schema for an error
                 allOf:
-                  - *ref_0
+                  - description: The response schema for a successful operation
+                    type: object
```

Comment:
It looks like our objects in the generated client code have questionable names like `inline_obect1`, `inline_object2`, `inline_object3`, ...etc. I wonder if there's something we can put in the spec to name them?

Reply:
some have meaningful names, some don't ... need to diff and check!